interface/modules/custom_modules/oe-module-weno/scripts/weno_log_sync.php

```diff
@@ -1,38 +1,62 @@
 <?php
 
 /**
- *  @package OpenEMR
- *  @link    http://www.open-emr.org
- *  @author  Sherwin Gaddis <sherwingaddis@gmail.com>
- *  @copyright Copyright (c) 2021 Sherwin Gaddis <sherwingaddis@gmail.com>
- *  @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ * @package    OpenEMR
+ * @link       http://www.open-emr.org
+ * @author     Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @author     Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright  Copyright (c) 2021 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright  Copyright (c) 2024 Jerry Padgett <sjpadgett@gmail.com>
+ * @license    https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  */
 
 use OpenEMR\Common\Crypto\CryptoGen;
+use OpenEMR\Common\Logging\EventAuditLogger;
 use OpenEMR\Modules\WenoModule\Services\LogProperties;
 use OpenEMR\Modules\WenoModule\Services\WenoPharmaciesJson;
 
 function downloadWenoPharmacy()
 {
     $cryptoGen = new CryptoGen();
-    $localPharmacyJson = new WenoPharmaciesJson(
-        $cryptoGen
-    );
+    $localPharmacyJson = new WenoPharmaciesJson($cryptoGen);
 
-    //check if the background service is active and set intervals to once a day
-    //Weno has decided to not force the import of pharmacies since they are using the iframe
-    //and the pharmacy can be selected at the time of creating the prescription.
+    // Check if the background service is active. Intervals are set to once a day
+    // Weno has decided to not force the import of pharmacies since they are using the iframe
+    // and the pharmacy can be selected at the time of creating the prescription.
     $value = $localPharmacyJson->checkBackgroundService();
+    EventAuditLogger::instance()->newEvent(
+        "pharmacy_background",
+        $_SESSION['authUser'],
+        $_SESSION['authProvider'],
+        3,
+        "Init Background Pharmacy Download Service Status:"  . text(ucfirst($value))
+    );
     if ($value == 'active' || $value == 'live') {
+        error_log('Background Initiated Pharmacy Download Started.');
+
         $status = $localPharmacyJson->storePharmacyDataJson();
-        error_log('Weno pharmacies download complete with status:' . text($status));
+
+        EventAuditLogger::instance()->newEvent(
+            "pharmacy_background",
+            $_SESSION['authUser'],
+            $_SESSION['authProvider'],
+            3,
```

Comment:
just checking why success setting is 3 in 2 calls to this function above

Reply:
experimenting.  really need to have event and category different names. not possible that I see. i.e pharmacy_background and pharmacy_download. will look closer at method.